.ci/install_cloud_hypervisor.sh

```diff
@@ -48,6 +46,9 @@ install_clh() {
 
 install_prebuilt_clh() {
 	local checksum_file="sha256sum-cloud-hypervisor"
+	go get -d "${go_cloud_hypervisor_repo}" || true
```

Comment:
I don't get why we need to clone the cloud hypervisor repo if we are only downloading a binary from our jenkins assets?

Reply:
@chavafg because of this https://github.com/kata-containers/tests/blob/master/.ci/ci_cache_components.sh#L63

Reply:
ok, thanks